lib/datawow.rb

```diff
@@ -13,6 +14,8 @@
 require_relative 'datawow/models/images/document_verification'
 require_relative 'datawow/models/predictions/predictors'
 
+require_relative 'datawow/models/moderations/moderation'
```

Comment:
I think only moderation
`require_relative 'datawow/models/moderations/moderation'` 
to
`require_relative 'datawow/models/moderation'`